R/Rules-methods.R

```diff
@@ -1776,6 +1791,15 @@ setMethod("stopTrial",
       ## so can we stop?
       doStop <- nPatients >= stopping@nPatients
 
+      report_label <- if (isTRUE(stopping@report_label == "")) {
+        paste(
+          "\u2265", x@nPatients, "patients in",
+          x@percentage, "% dose range around NBD"
```

Comment:
same here: what is the `x` here? I do not see this object on the list of explicit arguments to this function.